composables/myFetch.ts

```diff
@@ -0,0 +1,19 @@
+export const myFetch = async (request, opts = {}) => {
+    const config = useRuntimeConfig();
+    const baseURL = config.public.baseURL;
```

Comment:
A lot of these variables aren't needed. I think we can probably remove the baseURL var and just put it directly into the URL function. 

I think it's probably tidier to keep everything else though. Thanks Ruby

Reply:
done

Reply:
Thanks Ruby